### PR TITLE
[fix] sdio driver

### PR DESCRIPTION
**Pull request by cc-caixf**
when drive SD, sdio state need manul change.

### CODE REVIEW

**Comment by whj4674672** [comment]
@liuduanfei

**Comment by liuduanfei** [comment]
> @liuduanfei

@Guozhanxin 测试没有问题。

File: libraries/drivers/drv_sdio.c [hunk 1]
```diff
@@ -430,7 +430,8 @@ struct rt_mmcsd_host *sdio_host_create(struct stm32_sdio_des *sdio_des)
     host->private_data = sdio;
 
     /* ready to change */
-    mmcsd_change(host);
+    /* SD need manual change */
+    // mmcsd_change(host);
```
Comment (whj4674672) [inline libraries/drivers/drv_sdio.c:434]: 这里注释后会导致 WIFI 启动失败，触发 `(card != RT_NULL) assertion failed at function:sdio_io_rw_direct, line number:125`

Comment (whj4674672) [reply libraries/drivers/drv_sdio.c:434]: https://github.com/RT-Thread-Studio/sdk-bsp-stm32h750-realthread-artpi/pull/158

是我的这份 PR 导致的错误，已经提交新的 PR

https://github.com/RT-Thread-Studio/sdk-bsp-stm32h750-realthread-artpi/pull/181

**Pull request merged**